client/js/TemplateContext.js

```diff
@@ -7,7 +7,14 @@ module.exports = {
       currency: 'USD',
       minimumFractionDigits: 2
     } ),
-
+    capitalizeWords(string) {
+        var split = string.split(' ')
+        var label = this.CapitalizeFirstLetter(split[0]) + ' '
+        for (var i = 1; i < split.length; i ++) {
+            label += this.CapitalizeFirstLetter(split[i]) + ' '
+        }
+        return label.trim()
+    },
```

Comment:
Utilize `higher order` functions ( map, filter, forEach ) here:

```
capitalizeWords(string) {
   return string.split(/s+/)
     .map( word => this.CapitalizeFirstLetter(word) )
     .join(' ')
}
```

Reply:
Please note that identations should be used in multi-line command.  In a lot of modern shops, folks using automatic linting and code styling tools that everyone's code runs through in order to maintain a strict consistency.  If you have interest in helping me add something like that to our stack please let me know.